common/datatest-stable/src/runner.rs

```diff
@@ -22,19 +22,91 @@ struct TestOpts {
     filter: Option<String>,
     #[structopt(long = "exact")]
     /// Exactly match filters rather than by substring
-    filter_exact: bool,
+    exact: bool,
```

Comment:
This variable does not need to be changed, and the existing name is more descriptive IMO.

Reply:
filter-exact doesn't match the standard, and as cargo test commands can run multiple test executables, there would be an incompatible set of flags meaning neither "carg test -- --exact" nor "carg test -- --filter-exact" would complete as a successful run.   Exactly the problem that triggered this CL to deal with a non understood "--ignored" by the test executor.

Reply:
Note that the name of the variable here does not affect the CLI interface since the name of the flag is explicitly called made to be `--exact` in the attribute line:
`#[structopt(long = "exact")]`

Reply:
ahh, nice.

Reply:
Yeah its nice when you want your variables to be more descriptive (or slightly differently worded)